sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -157,8 +158,8 @@ public virtual async Task<Response<DetectedLanguage>> DetectLanguageAsync(string
                 if (languageDetection.Results.Errors.Count > 0)
                 {
                     // only one document, so we can ignore the id and grab the first error message.
-                    var error = Transforms.ConvertToError(languageDetection.Results.Errors.FirstOrDefault().Error);
-                    throw await _clientDiagnostics.CreateRequestFailedExceptionAsync(response, new ResponseError(error.ErrorCode.ToString(), error.Message), CreateAdditionalInformation(error)).ConfigureAwait(false);
+                    var error = languageDetection.Results.Errors.First().Error;
```

Comment:
I tripped over these after a rebase due to a name collision.   In looking at it, we didn't need to convert - we have the available information.  Since we already know there's at least one error, I also removed the `FirstOrDefault` in favor of just grabbing the first element.